plugins/coverbrowser.koplugin/bookinfomanager.lua

```diff
@@ -100,6 +101,7 @@ local BOOKINFO_COLS_SET = {
         "cover_bb_type",
         "cover_bb_stride",
         "cover_bb_data",
+        "identifiers"
```

Comment:
It should be inserted before the "cover" items, to be always accessible in `BookInfoManager:getBookInfo()`.

Reply:
Removed from cache db